### PR TITLE
Rel 0.101.4

### PR DESCRIPTION
**Pull request by AnnaShaleva**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,48 @@
 
 This document outlines major changes between releases.
 
+## 0.101.4 "Yarborough" (01 aug 2023)
+
+Another one 3.5.0-compatible version that is aimed to fix T5 testnet state
+difference happened at block 2336911 which leads to inability to process new
```
Comment (roman-khimov) [inline CHANGELOG.md:8]: that has happened

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,48 @@
...
+returned by an RPC server for error responses, invalid format of incremental
+dumps created by CLI and deadlock on unhealthy RPC server shutdown. Long-awaited
+`--config-file` CLI option to start the node providing a single configuration
+file is added.
+
+T5 testnet chain requires a complete resynchronization for this version. Mainnet
+chain resynchronization is recommended, but not required.
+
+New features:
+
```
Comment (roman-khimov) [inline CHANGELOG.md:24]: Why this empty line?

File: CHANGELOG.md [hunk 3]
```diff
@@ -2,6 +2,48 @@
 
 This document outlines major changes between releases.
 
+## 0.101.4 "Yarborough" (01 aug 2023)
```
[the remark this answers is not in the document]
Comment (AnnaShaleva) [reply CHANGELOG.md]: Fixed.

File: CHANGELOG.md [hunk 4]
```diff
@@ -2,6 +2,48 @@
...
+
+Another one 3.5.0-compatible version that is aimed to fix T5 testnet state
+difference happened at block 2336911 which leads to inability to process new
+blocks since 2418703. The issue is fixed by allowing JSON numbers unmarshalling
+from scientific notation to Integer stackitem. Maximum parsing precision for such
+numbers is currently restricted by 52 bits and will likely be increased in future
+releases.
```
[the remark this answers is not in the document]
Comment (AnnaShaleva) [reply CHANGELOG.md]: Then do we need this `Maximum parsing precision ...` sentence at all?

File: CHANGELOG.md (hunk 2 above)
Comment (AnnaShaleva) [reply CHANGELOG.md]: Fixed.

File: CHANGELOG.md [hunk 5]
```diff
@@ -2,6 +2,48 @@
...
+dumps created by CLI and deadlock on unhealthy RPC server shutdown. Long-awaited
+`--config-file` CLI option to start the node providing a single configuration
+file is added.
+
+T5 testnet chain requires a complete resynchronization for this version. Mainnet
+chain resynchronization is recommended, but not required.
+
+New features:
+
+* `--config-file` CLI option allowing to start the node with a single configuration file (#3014)
```
[the remark this answers is not in the document]
Comment (AnnaShaleva) [reply CHANGELOG.md]: Fixed.

File: CHANGELOG.md [hunk 6]
```diff
@@ -2,6 +2,48 @@
...
+* improper WSClient notification channels managing after disconnection (#2980)
+* missing Prometheus metric initialisation on node start (#2992)
+* invalid initialisation of native contracts cache (#2994)
+* incorrect way of incremental DB dumps creation (#3047)
+* Notary contract is allowed to be a sender of main Notary request transaction (#3065)
+* discrepancy in signer's witness scope parsing on the RPC server side (#3060)
+* Invoker calling API isn't allowed to accept nil parameter (#3067)
+* contract RPC Client unwrapper helper can't handle missing contract case (#3072)
+* JSON numbers can't be unmarshalled to stackitem from scientific notation (#3073)
+* invalid content-type header returned by RPC server on error responses (#3075)
```
[the remark this answers is not in the document]
Comment (AnnaShaleva) [reply CHANGELOG.md]: Fixed.

**Pull request merged**